cmd/lakectl/cmd/doctor.go

```diff
@@ -61,56 +61,82 @@ var errorTemplate = `{{ .Message |red }}
 var successMessageTemplate = `{{ .Message | green}}
 `
 
-var analayzingMessageTemplate = `{{ .Message }}
+var analyzingMessageTemplate = `{{ .Message }}
 `
 
 // doctorCmd represents the doctor command
 var doctorCmd = &cobra.Command{
 	Use:   "doctor",
 	Short: "Run a basic diagnosis of the LakeFS configuration",
+	Long:  "Checking correctness of access key ID, secret access key and endpoint URL",
 	Run: func(cmd *cobra.Command, args []string) {
-		err := ListRepositoriesAndAnalyze(cmd.Context())
+		verbose, _ := cmd.Flags().GetBool("verbose")
+		err := ListRepositoriesAndAnalyze(cmd.Context(), verbose)
 		if err == nil {
 			Write(successMessageTemplate, &UserMessage{"Valid configuration"})
 			return
 		}
 
+		if verbose {
```

Comment:
A thought: instead of using the 
```
if verbose {
...
}
```
clause, you can create a:
```
func WriteVerbose(tpl string, data interface{}, verbose bool) {
    if verbose {
        Write(tpl, data, os.Stdout)
    }
}
```
function that will print only if the verbose flag was given and use it instead.
I think it would make the code a bit cleaner.